web/assets/css/blog_post.css

```diff
@@ -21,14 +21,23 @@
 
 .blog-post-author {
     margin-top: 25px;
+}
+
+.blog-post-body:before {
```

Comment:
On other hand it could be achieved by `border-top` on the `blog-post-body`. I will update it